src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -433,12 +433,19 @@ void
 OverlayImpl::setupValidatorKeyManifests (BasicConfig const& config,
                                          DatabaseCon& db)
 {
-    auto const loaded = manifestCache_.loadValidatorKeys (
+    auto loaded = manifestCache_.loadValidatorKeys (
+        config.section ("validators"),
+        journal_);
+
+    if (!loaded)
+        Throw<std::runtime_error> ("Unable to load keys from [validators]");
+
```

Comment:
instead of duplicating the code here, what do you think about looping over the sections? Something like:
`for (std::string section : {"validators", "validator_key"})`